website/graphql/Connections.md

```diff
@@ -166,7 +166,11 @@ certain optimizations if this field returns an object that implements
 ### Cursor
 
 An "Edge Type" must contain a field called `cursor`. This field must return
-either a String or a Non-Null wrapper around a String.
+either a type that serializes as a String; this may be a String, a Non-Null
+wrapper around a String, or a custom scalar that serializes as a String.
```

Comment:
> I.e. it may be a String, a custom scalar that serializes as a String or a Non-Null wrapper around either.

That's correct, I'll update that to make that explicit.